public/app/features/dashboard-scene/scene/NavToolbarActions.tsx

```diff
@@ -578,6 +579,12 @@ export function ToolbarActions({ dashboard }: Props) {
     },
   });
 
+  toolbarActions.push({
+    group: 'new-export-dashboard-button',
+    condition: config.featureToggles.newDashboardSharingComponent && showShareButton,
+    render: () => <ExportButton key="new-export-dashboard-button" dashboard={dashboard} />,
+  });
+
```

Comment:
the export button is before Share button and is part of the same group so it doesn't have boundaries between each other (I think that's the group prop, so we should change to be the same)
![image](https://github.com/grafana/grafana/assets/11707172/dc44d3fa-ef05-4599-b332-06a344c35009)